frontend/src/components/modal/LoginModal.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Button,
+  Modal,
+  PasswordInput,
+  Stack,
+  Text,
+  TextInput,
+  Title,
+  UnstyledButton,
+} from '@mantine/core';
+import { isEmail, isNotEmpty, useForm } from '@mantine/form';
+
+interface LoginModalProps {
+  isLoginModalOpened: boolean;
+  closeLoginModal: () => void;
+  openSignUpModal: () => void;
+}
+
+function LoginModal({
+  isLoginModalOpened,
+  closeLoginModal,
+  openSignUpModal,
+}: LoginModalProps) {
+  const form = useForm({
+    mode: 'uncontrolled',
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validate: {
+      email: isEmail('Invalid email'),
+      password: isNotEmpty('Password cannot be empty'),
```

Comment:
might want to use custom mantime validation for this one... not empty is not super great for pw validation... might want to do both backend and frontend ux wise?